apps/web/package.json

```diff
@@ -76,6 +80,7 @@
     "react-test-renderer": "^18.3.1",
     "redux": "^5.0.1",
     "redux-persist": "^6.0.0",
+    "valtio": "^2.0.0",
```

Comment:
I am sure we don't need it. most likely you need to use redux that's already included